src/commands/commandui.cpp

```diff
@@ -164,6 +164,18 @@ int CommandUI::run(QStringList& tokens) {
       MockDaemon* daemon = new MockDaemon(qApp);
       qputenv("MVPN_CONTROL_SOCKET", daemon->socketPath().toLocal8Bit());
     }
+#ifdef MZ_IOS
+    else {
+      QString simDeviceName = QString(qgetenv("SIMULATOR_DEVICE_NAME"));
+      if (!simDeviceName.isEmpty()) {
```

Comment:
```suggestion
      bool isSimDevice = !QString(qgetenv("SIMULATOR_DEVICE_NAME")).isEmpty;
      if (isSimDevice) {
```

Nit: I find it more readable when it looks something like this (here and in other spots we check for sim). I'm a tad concerned a far-future reader will not immediately understand that this is checking just for iOS simulator (and not generally checking for iOS), but it's up to you.